### DISCUSSION TITLE
Can I execute a dag run with a date in the future?

### DISCUSSION THREAD

**Discussion by match-gabeflores** [apache/airflow#17655] [Q&A]
I want execute a dag with a date in the future. I went to Dag Runs and set the future date (9/1/2021). It's queued but it never starts running. Is this because it's in the future? Is this not possible in airflow?

Note, this dag runs daily in the morning without an issue. 

v1.10.15
![image](https://user-images.githubusercontent.com/286903/129754472-1205d383-a3ab-4fec-adcd-86de332bec74.png)

**Comment by ephraimbuddy** [comment]
Yes, you can. See the configuration doc: https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#allow-trigger-in-future

**Reply by match-gabeflores** [reply]
Thanks for that. But it says `Only has effect if schedule_interval is set to None in DAG`. Our schedule interval is set to `schedule_interval='0 9 * * *'`

I don't really understand this limitation of not being able to run in the future.
